### ISSUE TITLE
Event (input) dosen't work

### ISSUE THREAD

**Issue by pookdeveloper** [nbfontana/ngx-currency#89]
Event (input) dosen't work, Im actually use google chrome

````html
<input` type="text" (input)="calculatePrice(rowData)" currencyMask
                    [options]="{ prefix: ' ', thousands: '.', decimal: ',', max :'1000000'  }"
                    [(ngModel)]="rowData.amount">
````

Thanks

**Comment by TAharshalcarpenter** [comment]
I am facing this issue too. Chrome Version: Version 81.0.4044.129.

**Comment by mfabioe** [comment]
it works with key events though

**Comment by pookdeveloper** [comment]
> it works with key events though

You say keypress keyup etc.. ? Is not the same behaviour ..

**Comment by mfabioe** [comment]
> > it works with key events though
> 
> You say keypress keyup etc.. ? Is not the same behaviour ..

yes. (keyup), (keydown), (keypress) events work.
idk why input/change are not working

**Comment by TAharshalcarpenter** [comment]
@mfabioe Its because of this area:
```
@HostListener("input", ["$event"])
  handleInput(event: any) {
    // This Line ------------------
    if (this.isChromeAndroid()) {
      !this.isReadOnly() && this.inputHandler.handleInput(event);
    }
  }
```

On all other events it is `if(!this.isChromeAndroid())` 

https://github.com/nbfontana/ngx-currency/blob/master/src/currency-mask.directive.ts
Line 86

**Comment by didiokener** [comment]
I am facing the same issue.
Change event doesn't work. Is it because NG_VALUE_ACCESSOR?
My input:
`<input [options]="{align : 'left', prefix: 'R$ '}" 
  matInput formControlName="valor" currencyMask required
  (change)="test()">`

**Comment by mxmp210** [comment]
> 
> 
> I am facing the same issue.
> Change event doesn't work. Is it because NG_VALUE_ACCESSOR?
> My input:
> 
> ```
> <input [options]="{align : 'left', prefix: 'R$ '}" 
>   matInput formControlName="valor" currencyMask required
>   (change)="test()">
> ```

yes, indeed change or input doesn't work, first I thought it's angular 10 but it doesn't work in 9 too.

**Comment by giovannipds** [comment]
I'm facing the same, did you guys come up with something?

**Comment by giovannipds** [comment]
As far as I've tested here, I could bind `change`, `keypress` and `keyup`, but couldn't `keydown` neither `input`.

**Comment by giovannipds** [comment]
I was able to get a similar behavior with `keypress` and `blur` but, of course, it's not equal to `input` event, unfortunately.

**Comment by giovannipds** [comment]
With the updated version (`2.5.2`) I could use `keyup`, `keydown`, `keypress` but couldn't use `input` or `change`.

**Comment by Zwimber** [comment]
> The best alternative solution for me was to use `keyup`.

This is a pretty good alternative, thanks for pointing that out!

**Comment by ricardolerma** [comment]
Is this bug being solved? This is a very serious problem for componte's life.

**Comment by ozcanavtat** [comment]
I have solved this issue with setter/getter property which binded to ngModel of input. On setter I have handled valuechange event.

**Comment by NayaraGoulart** [comment]
I'm facing the same issue in version 3.0.0

**Comment by ArtemDintecom** [comment]
@Gabiruti The author does not develop new features, but accepts PR, although not quickly https://github.com/nbfontana/ngx-currency/pull/169#issuecomment-1636160301

**Comment by Alabarse** [comment]
<input type="text" class="rounded-2 w-50 h-100 form-control text-end"
               [(ngModel)]="valorAcrescimo" (input)="getTotalOuParcial(totalOuParcial)" currencyMask>

When i was using the input event with 'currencyMask' the getTotalOuParcial() method has not been called, but when i removed the currency mask it works perfectly

**Comment by ArtemDintecom** [comment]
@Alabarse You could use the ngModelChange event or turn valorAcrescimo into a get set property
